langchain-llamaindex-slackbot/src/chat_lanarky.py

```diff
@@ -71,9 +79,7 @@
 model = "openai"
 
 if model == "openai":
-    openai_llm = OpenAI(
-        temperature=0, max_tokens=1000, model_name="gpt-3.5-turbo"
-    )
+    openai_llm = OpenAI(temperature=1, max_tokens=500, model_name="gpt-4")
```

Comment:
1 is actually the default and is what OpenAI recommend. I think we set it to 0 when we first set this up to be extra conservative but since then I've worked with it quite a bit and 1 is fine, I think. Above 1 is when you start to get weird responses + too much out of the bounds of what is reasonable.